src/Integration.Vsix/SonarLint.vsct

```diff
@@ -46,17 +47,13 @@
           <ButtonText>Help</ButtonText>
         </Strings>
       </Menu>
-    </Menus>
-
-    <!-- Example of how to add a command to the sonarlint sub menu
```

Comment:
I removed this commented sample because we're already using it, so we have the real buttons as examples.